README.md

```diff
@@ -308,7 +298,7 @@ $ ./bin/occollector_$($GOOS)
 (note: additional ports may be required depending on your receiver configuration):
 ```shell
 $ make docker-collector
-$ docker run --rm -it -p 55678:55678 occollector
+$ docker run --rm -it -v $(pwd)/occollector-config.yaml:/conf/occollector-config.yaml -p 55678:55678 -p 8888:8888 occollector
```

Comment:
You need to pass `--config=/conf/occollector-config.yaml`

Reply:
fixing